packages/utils-test/src/setupFixture.ts

```diff
@@ -28,5 +30,8 @@ export async function setupFixture(fixtureName: string) {
   fs.mkdirpSync(cwd);
   fs.copySync(fixturePath, cwd);
 
+  await execa("git", ["init"], { cwd });
```

Comment:
Is it faster/better to replace this with just generating a .git folder? It doesn't look like it affects the test speed that much from what I could see, but just a thought.

Reply:
right